src/components/Dashboard/DonationHistories/DonationHistories.tsx

```diff
@@ -242,15 +238,23 @@ const DonationHistories = ({
                     >
                       <Legend />
                       <CartesianGrid vertical={false} />
-                      {goal && (
+                      {data?.healthIndicatorData?.length === 0 ? (
```

Comment:
Did you make it `=== 0` to make it more readable that it' has no length?
```suggestion
                      {!data?.healthIndicatorData?.length ? (
```